src/lib/util.ts

```diff
@@ -231,6 +231,18 @@ export function parseToNumberString(num: number, getString: GetStringFunction):
 	return `${num}`
 }
 
+export const locales = readdirSync("./strings")
```

Comment:
This reduces IO operations later down the line, because I made other files use this so they don't call readdirSync every time an event is emitted.

Reply:
Readdir is very quick and you do not need to worry about it taking up resources because it really doesn't. Much rather have that being ran when it's needed than on startup

Reply:
Eh, ok, I can revert this change.